examples/python_boids_spatial3D_wrapped_native/model/Attribution.txt

```diff
@@ -0,0 +1 @@
+The 3D bird models (bird_a.obj and bird_b.obj) are key frames extracted from "Flying Bird" 3D model (https://sketchfab.com/3d-models/flying-bird-eb843194e06d429ebef7dd4aa7e265c1) by "sandeep.s" licenced under CC-BY 4.0 (https://creativecommons.org/licenses/by-nd/2.0/)
```

Comment:
> extracted from "Flying Bird"

Then you link to a "No Derivatives license". You can't modify it and redistribute.

P.S. You state CC BY 4.0, but link to CC BY-ND 2.0

Reply:
(This is why for the "stuntplane" stock model I just spent 30 minutes following a blender tutorial. So it would be free from any licenses.

Reply:
Appears if you click the license thing on sketchfab it takes you to `https://creativecommons.org/licenses/by/4.0/`, so it's not the ND version but you should fix the link.

```suggestion
The 3D bird models (bird_a.obj and bird_b.obj) are key frames extracted from "Flying Bird" 3D model (https://sketchfab.com/3d-models/flying-bird-eb843194e06d429ebef7dd4aa7e265c1) by "sandeep.s" licenced under CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)
```

Reply:
Not sure how I did this but have fixed this.